src/pharmpy/modeling/remove_iov.py

```diff
@@ -4,16 +4,25 @@
 
 import warnings
 
+from pharmpy.modeling import remove_unused_parameters_and_rvs
+from pharmpy.modeling.help_functions import _format_input_list
 from pharmpy.random_variables import RandomVariables
+from re import search
 
 
-def remove_iov(model):
+def remove_iov(model, to_remove_parameters=None, to_remove_symbols=None):
     """Removes all IOV etas
 
     Parameters
     ----------
     model : Model
         Pharmpy model to remove IOV from.
+    to_remove_parameters : str, list
+        Name/names of omega parameters for IOV etas to remove, e.g. 'OMEGA_IOV_1'.
+        If None, all etas that are IOVs and related parameters will be removed. None is default.
+    to_remove_symbols : str, list
```

Comment:
The IOV assignments are really arbitrary. I think this option should be removed.

Reply:
> The IOV assignments are really arbitrary. I think this option should be removed.

Ok, I can delete this option!